CONTRIBUTING.md

```diff
@@ -8,13 +8,13 @@ If there isn't one, [create one](https://github.com/Tadaboody/good_smell/issues/
 If you want to contribute code you will need to:
 
 ## Create an enviroment
-This project uses [Pipenv](https://github.com/pypa/pipenv) for package managment
+This project uses [Poetry](https://python-poetry.org/) for package managment
```

Comment:
personally, i would link to the docs here. my first impression of the site isn't great, and the contrast on the link to the documentation isn't great; might be hard for someone to find. just my opinion though